vou/batch.py

```diff
@@ -30,33 +30,33 @@ def simulate(self, parallel: bool = True):
         generator, allowing for reproducibility.
         """
 
-        def simulate_one_person(self, seed: int):
+        def simulate_one_person(self, param_dict: dict):
             person = Person(
-                rng=Random(seed),
-                starting_dose=self.params["starting_dose"],
-                dose_increase=self.params["dose_increase"],
-                external_risk=self.params["external_risk"],
-                internal_risk=self.params["internal_risk"],
-                behavioral_variability=self.params["behavioral_variability"],
+                rng=Random(param_dict['seed']),
```

Comment:
It would be better to instantiate `rng=Random(seed)` once at the start of `simulate_one_person`, then call `rng=rng` for both Person and Simulation. (This isn't something you changed, I just happened to notice it now.)